API.md

```diff
@@ -1764,12 +1764,23 @@ Requires the string value to be a valid hexadecimal string.
 const schema = Joi.string().hex();
 ```
 
-#### `string.base64()`
+#### `string.base64([padding])`
```

Comment:
I can, but the only other options I can anticipate would be around decoding? Trying to be cognizant of over-engineering.